app/src/main/java/com/github/swent/echo/compose/components/HamburgerMenuDrawerSheet.kt

```diff
@@ -95,17 +96,17 @@ fun HamburgerMenuDrawerSheet(
                 title = stringResource(id = R.string.hamburger_add_friends),
                 selectedIcon = Icons.Filled.Add,
             ),
-            NavigationItem(
-                title = stringResource(id = R.string.hamburger_help),
-                selectedIcon = Icons.Filled.Build,
-            )
-
              */
             NavigationItem(
                 title = stringResource(R.string.hamburger_associations),
                 selectedIcon = Icons.Filled.Star,
                 navOnClick = { navActions.navigateTo(Routes.ASSOCIATIONS) }
             ),
+            NavigationItem(
+                title = stringResource(id = R.string.hamburger_help),
+                selectedIcon = Icons.Filled.Build,
```

Comment:
another icon would make more sense in my opinion, I suggest [the "help" icon from google](https://fonts.google.com/icons?icon.platform=android&selected=Material+Symbols+Outlined:help:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=help)